autogpt_platform/backend/backend/server/integrations/router.py

```diff
@@ -208,7 +218,67 @@ def delete_credentials(
     return CredentialsDeletionResponse(revoked=tokens_revoked)
 
 
-# -------- UTILITIES --------- #
+# ------------------------- WEBHOOK STUFF -------------------------- #
+
+
+@router.post("/{provider}/webhooks/{webhook_id}/ingress")
+async def webhook_ingress_generic(
+    request: Request,
+    provider: Annotated[
+        ProviderName, Path(title="Provider where the webhook was registered")
+    ],
+    webhook_id: Annotated[str, Path(title="Our ID for the webhook")],
+):
```

Comment:
Comment why this has no user id, and add a follow up for API key

Reply:
> and add a follow up for API key

? WDYM?